### DISCUSSION TITLE
Are signers of this petition being blacklisted?

### DISCUSSION THREAD

**Discussion by steveeq1** [rms-support-letter/rms-support-letter.github.io#6749] [General]
I read somewhere on reddit that signers of this petition are being blacklisted by companies? Is this true?

I'm not ashamed of my position and stand by it, but I find it interesting that people would go that out of their way to do something like that over something so petty.

**Comment by nukeop** [comment]
No, it's not true.

**Comment by Aspie96** [comment]
I've read of people saying to use this list as a non hiring list so I'd say it's probably true..

**Reply by nukeop** [reply]
Most of those people are teenagers and other people who have no power over hiring decisions. Don't believe everything you read on the internet, especially posturing.

**Reply by Aspie96** [reply]
I sure hope you are correct.

**Comment by 6r1d** [comment]
> I'm not ashamed of my position and stand by it

In general, this is reaffirming. If some group tries to intimidate a single person, they might succeed. If someone tries that on thousands... It won't work very well, would it now?

> but I find it interesting that people would go that out of their way to do something like that over something so petty.

It's called "control freaks", but you _generally don't want to have anything to do with them in the first place_.
Let them reassure each other, just don't feed the fire by paying attention to them.

**Comment by cmpunches** [comment]
Not where I can see it.

**Reply by 6r1d** [reply]
Good to hear, @cmpunches
